Makefile

```diff
@@ -1,321 +1,259 @@
+	go mod tidy
+	go fmt ./...
 
-# Build/install/release the SDK.
-.PHONY: release_builds release
+.PHONY: clean
+clean: ## Cleanup build artifacts and tool binaries.
+	rm -rf $(BUILD_DIR) dist tools/bin
 
-release_builds := \
-	build/operator-sdk-$(GIT_VERSION)-aarch64-linux-gnu \
-	build/operator-sdk-$(GIT_VERSION)-x86_64-linux-gnu \
-	build/operator-sdk-$(GIT_VERSION)-x86_64-apple-darwin \
-	build/operator-sdk-$(GIT_VERSION)-ppc64le-linux-gnu \
-	build/operator-sdk-$(GIT_VERSION)-s390x-linux-gnu \
-	build/ansible-operator-$(GIT_VERSION)-aarch64-linux-gnu \
-	build/ansible-operator-$(GIT_VERSION)-x86_64-linux-gnu \
-	build/ansible-operator-$(GIT_VERSION)-x86_64-apple-darwin \
-	build/ansible-operator-$(GIT_VERSION)-ppc64le-linux-gnu \
-	build/ansible-operator-$(GIT_VERSION)-s390x-linux-gnu \
-	build/helm-operator-$(GIT_VERSION)-aarch64-linux-gnu \
-	build/helm-operator-$(GIT_VERSION)-x86_64-linux-gnu \
-	build/helm-operator-$(GIT_VERSION)-x86_64-apple-darwin \
-	build/helm-operator-$(GIT_VERSION)-ppc64le-linux-gnu \
-	build/helm-operator-$(GIT_VERSION)-s390x-linux-gnu
+##@ Build
 
-release: clean $(release_builds) $(release_builds:=.asc) ## Release the Operator SDK
+.PHONY: build
+build: ## Build operator-sdk, ansible-operator, and helm-operator.
+	mkdir -p $(BUILD_DIR) && go build $(GO_BUILD_ARGS) -o $(BUILD_DIR) ./cmd/operator-sdk ./cmd/ansible-operator ./cmd/helm-operator
 
-build/operator-sdk-%-aarch64-linux-gnu: GOARGS = GOOS=linux GOARCH=arm64
-build/operator-sdk-%-x86_64-linux-gnu: GOARGS = GOOS=linux GOARCH=amd64
-build/operator-sdk-%-x86_64-apple-darwin: GOARGS = GOOS=darwin GOARCH=amd64
-build/operator-sdk-%-ppc64le-linux-gnu: GOARGS = GOOS=linux GOARCH=ppc64le
-build/operator-sdk-%-s390x-linux-gnu: GOARGS = GOOS=linux GOARCH=s390x
-build/operator-sdk-%-linux-gnu: GOARGS = GOOS=linux
-
-build/ansible-operator-%-aarch64-linux-gnu: GOARGS = GOOS=linux GOARCH=arm64
-build/ansible-operator-%-x86_64-linux-gnu: GOARGS = GOOS=linux GOARCH=amd64
-build/ansible-operator-%-x86_64-apple-darwin: GOARGS = GOOS=darwin GOARCH=amd64
-build/ansible-operator-%-ppc64le-linux-gnu: GOARGS = GOOS=linux GOARCH=ppc64le
-build/ansible-operator-%-s390x-linux-gnu: GOARGS = GOOS=linux GOARCH=s390x
-build/ansible-operator-%-linux-gnu: GOARGS = GOOS=linux
-
-build/helm-operator-%-aarch64-linux-gnu: GOARGS = GOOS=linux GOARCH=arm64
-build/helm-operator-%-x86_64-linux-gnu: GOARGS = GOOS=linux GOARCH=amd64
-build/helm-operator-%-x86_64-apple-darwin: GOARGS = GOOS=darwin GOARCH=amd64
-build/helm-operator-%-ppc64le-linux-gnu: GOARGS = GOOS=linux GOARCH=ppc64le
-build/helm-operator-%-s390x-linux-gnu: GOARGS = GOOS=linux GOARCH=s390x
-build/helm-operator-%-linux-gnu: GOARGS = GOOS=linux
-
-build/%: $(SOURCES) ## Build the operator-sdk binary
-	$(Q){ \
-	cmdpkg=$$(echo $* | sed -E "s/(operator-sdk|ansible-operator|helm-operator).*/\1/"); \
-	$(GOARGS) go build $(GO_BUILD_ARGS) -o $@ ./cmd/$$cmdpkg; \
-	}
+.PHONY: install
+GOBIN ?= $(shell go env GOPATH)/bin
```

Comment:
```suggestion
GOBIN ?= $(shell go env GOBIN)
```

Reply:
In my environment, `go env GOBIN` is empty, whereas `go env GOPATH` is set to `/home/joe/go`, even though I haven't explicitly set `GOPATH`.

On second thought, I'm just going to revert this target back to use `go install` since that should take care of all the details for us.

Reply:
I was under the impression that this was always set even if you never `export GOBIN=path`, but I don't think that's true. :+1: to using `go install` behavior.